### ISSUE TITLE
Compatibility with Self Hosted Wordpress Sites

### ISSUE THREAD

**Issue by jonaharagon** [waltzio/waltz#324]
I would like to request support Waltz compatibility with Wordpress Self Hosted sites, hopefully through a plugin.

The websites in particular I would like compatibility with are: http://paracord.ml, http://mvgov.pw, http://writers.mvgov.pw, and http://beta.mcinternet.tk.

**Comment by jessepollak** [comment]
Have you seen the Clef WordPress plugin? You can find it here:
http://wordpress.org/plugins/wpclef.

**Comment by jonaharagon** [comment]
Yes, but it would be nice if it would just be linked with Waltz. For now I am offering Clef on most of those sites.

**Comment by jessepollak** [comment]
Hmm, what would be the benefit of using Waltz instead of the plugin? That you wouldn't have to install it on the sites?

**Comment by jonaharagon** [comment]
Yes, and that if you use Waltz to login to say, Google, you'll also be logged into the WordPress site. It would just make it easier for end-users. Those sites mostly don't apply anymore, but I would still like this to happen somehow for other sites, even if it is through a plugin, just because of the connection.
